packages/common/driver-definitions/src/storage.ts

```diff
@@ -298,6 +299,16 @@ export interface IDocumentDeltaConnection
 	// TODO: Use something other than `any`.
 	// eslint-disable-next-line @typescript-eslint/no-explicit-any
 	submitSignal(content: any, targetClientId?: string): void;
+
+	/**
+	 * Tells whether the connection supports connection updates events on the connection object
+	 */
+	supportDeltaConnectionUpdates?: boolean;
```

Comment:
should we add "readonly" attribute to this and other properties?

Reply:
Removed it since no longer needed after emitting event on service.